src/consts.js

```diff
@@ -102,6 +103,7 @@ module.exports = {
         windowCmdWebContentsCloseDevTools: "window.cmd.web.contents.close.dev.tools",
         windowCmdWebContentsOpenDevTools: "window.cmd.web.contents.open.dev.tools",
         windowCmdWebContentsExecuteJavascript: "window.cmd.web.contents.execute.javascript",
+        windowEventAlwaysOnTopChanged: "window.event.always.on.top.changed",
```

Comment:
Could you rename the event to `window.event.always.on.top.set` as well as the const to `windowEventAlwaysOnTopSet` ?

Reply:
Ok, sure.
Although the way I implemented it was based on the official Electron docs.

Reply:
@asticode 

Here's the documentation for that event.
Is there a reasoning for wanting to differ from the reference implementation?

https://www.electronjs.org/docs/latest/api/browser-window#event-always-on-top-changed

Reply:
IMHO, `changed` is preferable to `set` for multiple reasons.

1. It's in the reference implementation (Electron).
2. It's less ubiquitous. `set` implies the existence of an `unset` but there is only one event.

Reply:
fair enough 👍